package.json

```diff
@@ -27,6 +27,7 @@
 		"build:gendocs:client:packages": "copyfiles \"packages/**/*.api.json\" ./_api-extractor-temp/doc-models/ -e \"**/node_modules/**\" -f -V",
 		"build:gendocs:server": "concurrently \"npm:build:gendocs:server:*\"",
 		"build:gendocs:server:routerlicious": "copyfiles \"server/routerlicious/**/*.api.json\" ./_api-extractor-temp/doc-models/ -e \"**/node_modules/**\" -f -V",
+		"build:readme": "markdown-magic --files \"**/*.md\" !docs",
```

Comment:
Should we remove this step from the `docs` build now that we have it here?

Reply:
I thought about that, but since the docs build imports stuff from elsewhere in the repo, seemed good to leave it as a step in the "docs build."